tests/test_actions.py

```diff
@@ -4,29 +4,29 @@
 from unittest import mock
 
 import pytest
+from conftest import mock_settings
+from unittest_protocols import ErrorQuestion
+from useful_mocks import (
+    get_mock_action_class,
+    get_mock_question_class,
+    get_mock_resolution_class,
+)
```

Comment:
I'm not sure if you did this on purpose, `isort` should have put these back down where they belong. If `isort` did this to you, then we need to fix its configuration. :P

Reply:
Yeah, I noticed isort moved that around.  It surprised me too.

Reply:
But I'm curious why they need to be down towards the bottom?

Reply:
I try to follow the pattern of "farthest to closest" to the code for import order (and then alphabetize). So at the top is the standard library stuff, the next section is third party things we're using, the next level is stuff in the same package/codebase that we're importing, and the bottom part is stuff within the same folder.

In other words, these shouldn't be up by `pytest`. I think we just need to tell `isort` that they're first party packages. :P

Reply:
Ah, i think we just need to add `tests` to `src_paths` on line 6 of `.isort.cfg`